internal/start/start.go

```diff
@@ -280,7 +282,25 @@ EOF
 			}
 			env = append(env, "DOCKER_HOST="+dindHost.String())
 		case "npipe":
-			fmt.Fprintln(os.Stderr, utils.Yellow("WARNING:"), "analytics requires docker daemon exposed on tcp://localhost:2375")
+			const dockerDaemonNeededErr = "Analytics requires docker daemon exposed on tcp://localhost:2375. Either disable analytics in supabase/config.toml, or expose the docker daemon. See https://supabase.com/docs/guides/local-development/cli/getting-started?queryGroups=platform&platform=windows#running-supabase-locally for more details"
+
+			resp, err := http.Get("http://localhost:2375/version")
```

Comment:
If docker host is already setup for `npipe` there's probably no need to call the tcp port.

IMO, it would be better to disable analytics here and let supabase start run to completion.

Reply:
Yes you are right my bad, the `tcp` port check will always fail because its already detected as `npipe` my bad

As for the suggestion regarding disabling analytics, I have a few ideas on how to implement it

Option A:
1. When installing supabase
2. Check if daemon running on `npipe`
3. If so, ask user if we want to disable analytics in the generated config
4. If yes, generate `config.toml` with `analytics=false`

Option B:
1. When running `supabase start`
2. Check if vector is failing
3. If vector is failing, then print the error, but continue allowing supabase to spin up fully

Let me know which one you think think is better and I can go ahead and do it. Thanks!

Reply:
Does it work on windows currently if you pass in `supabase start --ignore-health-check` flag? If so, then option B sounds like a good solution.

Reply:
Basically changing the following lines https://github.com/supabase/cli/blob/develop/internal/start/start.go#L331

```golang
		if parsed.Scheme != "npipe" {
			started = append(started, utils.VectorId)
		}
```